### PR TITLE
feat: utility script for generating static routes file

### PR DESCRIPTION
**Pull request by dohaki**
The static `routes.json` files became hard to maintain with the recent additions of new chains/tokens. This PR adds a script to make future additions/removals easier.

### CODE REVIEW

File: scripts/generate-routes.ts [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+          },
+          {
+            chainId: CHAIN_IDs.BASE,
+            tokens: ["WETH", "ETH", "USDC", "DAI", "BAL"],
+          },
+        ],
+      },
+      {
+        fromChain: CHAIN_IDs.POLYGON,
+        fromSpokeAddress: getDeployedAddress("SpokePool", CHAIN_IDs.POLYGON),
```
Comment (james-a-morris) [inline scripts/generate-routes.ts:160]: Did we want to expose this for all chain_IDs?

File: scripts/generate-routes.ts [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+      address: "0x269727F088F16E1Aea52Cf5a97B1CD41DAA3f02D",
+      enabledChains: [
+        CHAIN_IDs.MAINNET,
+        CHAIN_IDs.OPTIMISM,
+        CHAIN_IDs.POLYGON,
+        CHAIN_IDs.BASE,
+        CHAIN_IDs.ARBITRUM,
+      ],
+    },
+    routes: [
```
Comment (james-a-morris) [inline scripts/generate-routes.ts:39]: This routes object feels like it could go into the constants-v2 repo. Thoughts on moving this there?

**Review by james-a-morris — COMMENTED** [review]
Looks great!

File: scripts/generate-routes.ts (hunk 1 above)
Comment (dohaki) [reply scripts/generate-routes.ts:160]: Sorry, I don't fully understand the comment 🤔 Can you elaborate a bit more?

File: scripts/generate-routes.ts (hunk 2 above)
Comment (dohaki) [reply scripts/generate-routes.ts:39]: Hmm, not so sure about that tbh. Doesn't this just add more friction when trying to update/remove/add routes ad-hoc? AFAICT the FE is also the only place where this is currently used. So I don't see any good reason to move it to the contracts-v2 repo

File: scripts/generate-routes.ts (hunk 1 above)
Comment (james-a-morris) [reply scripts/generate-routes.ts:160]: Sorry later on in the code is I see we handle this. I misunderstood the code file.

**Review by james-a-morris — APPROVED** [review]
LGTM

**Pull request merged**